src/prettytable/prettytable.py

```diff
@@ -448,6 +448,16 @@ def _validate_attributes(self, name, val):
     # ATTRIBUTE MANAGEMENT       #
     ##############################
 
+    @property
+    def xhtml(self):
+        """Print <br/> tags if True, <br> tags if false"""
```

Comment:
```suggestion
        """Print <br/> tags if True, <br> tags if False"""
```
? And in the other place this text is used?

Reply:
Fixed the docstring.

Regarding your first question - 
Yes this change makes the code more consistent with its design. The "xhtml" option is the only option that is not exposed as a property, and I believe it could be useful.
This consistency allows implementing #104 in a more elegant way.